pages/linux/agetty.md

```diff
@@ -0,0 +1,29 @@
+# agetty
+
+> Alternative `getty`: Open a `tty` port, prompt for a login name, and invoke the `/bin/login` command.
+> It is normally invoked by `init`.
+> More information: <https://manned.org/agetty>.
+
+- Connect `stdin` to a port (relative to `/dev`) and optionally specify a baud rate (defaults to 9600):
```

Comment:
What's a baud rate? I think that `baud` is a very uncommon word. This description would profit from explaining what this means.

Reply:
Baud rate is a very common concept whenever serial comm is involved. Examples include: MCU programming/debugging, enterprise network management via console port, etc.

For the intended target audience of this tool, I don't think it's necessary to explain the term.

Reply:
I disagree, IMO it doesn't hurt to briefly explain this term in the description. Also, a not so new user can discover this command not knowing what is a baud rate.

See the second guideline from CONTRIBUTING.md:
> When in doubt, keep new command-line users in mind. Err on the side of clarity rather than terseness. For example, commands that require sudo should include it directly in the examples.